### ISSUE TITLE
New case form not loading correctly

### ISSUE THREAD

**Issue by campgurus** [EBWiki/EBWiki#1275]
This is what I see in the form field for agencies when I initially load the form:

<img width="820" alt="screen shot 2018-02-14 at 7 20 35 am" src="https://user-images.githubusercontent.com/1260659/36204029-ac554980-1157-11e8-9463-023102c45255.png">

On refresh, the javascript seems to load.  Are others seeing this?

**Comment by rlgreen91** [comment]
Are you able to reproduce this?  Do you know what the logs say when you see this?

**Comment by campgurus** [comment]
The logs don't say anything notable.  There is no error. But for some reason, the collection field does not pull the list of agencies. The only choices are "Yes" and "No". Refreshing the form does not alleviate the issue. But returning to the form via the "edit" action pulls in the list of agencies.  It make me think that there is something in the controller.

here is the trace:
```
2018-02-27T10:35:50.830440+00:00 app[web.1]: Started GET "/articles/new" for 162.158.62.232 at 2018-02-27 10:35:50 +0000
2018-02-27T10:35:50.832715+00:00 app[web.1]: Processing by ArticlesController#new as HTML
2018-02-27T10:35:50.859958+00:00 app[web.1]:   Rendered articles/_subject_fields.html.erb (13.0ms)
2018-02-27T10:35:50.875925+00:00 app[web.1]:   Rendered articles/_link_fields.html.erb (2.2ms)
2018-02-27T10:35:50.877853+00:00 app[web.1]:   Rendered articles/_form.html.erb (39.3ms)
2018-02-27T10:35:50.878076+00:00 app[web.1]:   Rendered articles/new.html.erb within layouts/application (39.8ms)
2018-02-27T10:35:50.889672+00:00 app[web.1]:   Rendered layouts/_header.html.erb (10.0ms)
2018-02-27T10:35:50.891214+00:00 app[web.1]:   Rendered layouts/_footer.html.erb (0.4ms)
2018-02-27T10:35:50.891470+00:00 app[web.1]: Completed 200 OK in 59ms (Views: 49.7ms | ActiveRecord: 4.6ms)
```

**Comment by campgurus** [comment]
The difference I see is in the edit action we have:

`@agencies = Agency.all.sort_by { |e| ActiveSupport::Inflector.transliterate(e.name.downcase) }`

In the new action, that variable is not assigned.

**Comment by rlgreen91** [comment]
Will have some time to tackle this later this week.

**Comment by rlgreen91** [comment]
Ok, so this issue is now the following: within the `new` method of `articles_controller`, set the `@agencies` value the same way as done in the `edit` method.

**Comment by rlgreen91** [comment]
Addressed in PR #1404
